### PR TITLE
feat: Success notification unlockable drop

### LINKED ISSUE #6204: Success notification unlockable drop
image
### main
- the thing will be shown when you click the mint button 
- there is a copy saying that the nft will be visible in an estimated 30 seconds 
- once the 30 seconds pass, the button with 3 dots animation and copy "preparing your item..." will change to "view item" 
- button redirects the user to the nft detail. 
- there is also an option to close this notification 
- should pop up in the middle of the screen - with darkened background (same as with the sidebar) 
- clicking outside the modal should not close the notification since I personally always accidentally close it :D 
- handoff file is updated - page unlockable drop page: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=589%3A6842&t=tGllyxKdl4QGJTHF-1

cc @vikiival I believe he has begun working on something?
- #6163

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #6204
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed UI

image

image

### <samp>🤖 Generated by Copilot at 2f58564</samp>

Added a new `UnlockableLoader` component to improve the user experience when minting and viewing unlockable NFTs. Replaced the old `Loader` component with the new one in the `UnlockableContainer` component. Used some components and hooks from the `@kodadot1/brick` library.

### <samp>🤖 Generated by Copilot at 2f58564</samp>

> _To mint NFTs with a flair_
> _We added a new `UnlockableLoader` there_
> _It uses some hooks and props_
> _And some `Neo` components on top_
> _And shows a congrats message with care_

### CODE REVIEW

**Comment by Jarsen136** [comment]
> Great work! Is there a way to activate the popup without minting the NFT? Could you guide me to a resource where I can find more information on this? Thanks!

Yes, I have made a preview env for testing. Opening the page with parms `?redesign=true`, the popup would be active auto and then the button would be changed after a few seconds.
`https://deploy-preview-6213--koda-canary.netlify.app/stmn/unlockable?redesign=true`.

I will remove this testing logic before the PR is about to merge.

File: components/collection/unlockable/UnlockableLoader.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+)
+const { urlPrefix } = usePrefix()
+const isLoading = useVModel(props, 'value')
+const emit = defineEmits(['input'])
+
+const closeLoading = () => {
+  emit('input', false)
+}
+const buttonLabel = computed(() =>
+  props.minted ? 'View Item' : 'Preparing Your Item...'
```
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLoader.vue:78]: i18n

**Review by roiLeo — COMMENTED** [review]
Text look a bit small on figma

Why don't we unify component between these two

Capture d’écran 2023-06-14 à 8 41 39 AM
Capture d’écran 2023-06-14 à 8 41 29 AM

**Comment by exezbcz** [comment]
@roiLeo its a different success, the one for drop page is in figma drop page

File: components/collection/unlockable/UnlockableLoader.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
...236.508 34.7398 235.198 35.7007 234.433 36.1507C232.586 37.2354 230.062 37.6803 227.505 37.3691C222.199 36.7238 216.951 37.738 212.723 40.2229C208.014 42.99 204.971 47.4081 204.374 52.3428C203.185 62.1707 211.483 70.9896 223.265 72.4212C223.521 72.4526 223.776 72.4677 224.028 72.4677C227.154 72.4677 229.86 70.1271 230.245 66.9386C230.662 63.4927 228.212 60.3607 224.774 59.9432C219.875 59.3483 216.537 56.2382 216.825 53.8556C216.992 52.4765 218.302 51.5156 219.068 51.0656C220.914 49.9809 223.439 49.536 225.995 49.8472C227.038 49.9742 228.078 50.0364 229.109 50.0364C233.323 50.0364 237.38 48.9903 240.777 46.9934C245.486 44.2263 248.529 39.8082 249.126 34.8735C249.723 29.9377 247.821 24.9191 243.908 21.1043ZM441.684 130.804C444.58 128.901 445.389 125.005 443.489 122.102C422.471 89.9844 438.585 67.3448 448.212 53.8185C449.383 52.1736 450.489 50.6194 451.391 49.2258C453.276 46.3138 452.449 42.4218 449.542 40.5329C446.636 38.644 442.753 39.4734 440.868 42.3854C440.117 43.5468 439.136 44.9241 438 46.5198C428.028 60.5313 407.277 89.6872 433 128.996C433.571 129.869 434.35 130.586 435.266 131.081C436.183 131.577 437.209 131.836 438.25 131.834C439.471 131.835 440.665 131.477 441.684 130.804Z"
```
Comment (vikiival) [inline components/collection/unlockable/UnlockableLoader.vue:20]: Isn't there some nicer way how to load svgs?

Comment (Jarsen136) [reply components/collection/unlockable/UnlockableLoader.vue]: Yes, there is

**Review by roiLeo — APPROVED** [review]
in anticipation of a unified design, let's roll

**Comment by exezbcz** [comment]
okay, a few minor changes 
- make the background the same as with opening the wallet connect sidebar - if its already, my bad :D 
- The button should be 16px font size 
- make the box close only with the close icon because otherwise, it could happen you accidentally close it. 

Thanks! 

image

**Review by vikiival — APPROVED** [review]

**Comment by exezbcz** [comment]
> It's already there.
> 
> >

Cool, I think I will create one more issue for some small things, I don't want to hold up this one. Thanks! 

visual oki 🏁

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.74 USD/DOT ~ 6.329 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x84c23fa2b4fd20e12a49840de308b4091c5f695e6dfe94c0cd65a186e633b6a6](https://polkadot.subscan.io/extrinsic/0x84c23fa2b4fd20e12a49840de308b4091c5f695e6dfe94c0cd65a186e633b6a6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**